backend/src/main/scala/cromwell/backend/ExecutionHash.scala

```diff
@@ -0,0 +1,13 @@
+package cromwell.backend
+
+import java.util.UUID
+
+import scala.concurrent.{ExecutionContext, Future}
+
+object ExecutionHash {
+  // TODO: PBE: ideally hashes should be deterministic
```

Comment:
At the very least, can we nix the `PBE: `, the PBE TODOs should either be TODONE, TODO or removed at this point